components/TextInput/TextInput.js

```diff
@@ -0,0 +1,16 @@
+import { Label } from "./StyeldTextInput";
+
+export default function TextInput({ id, name, value, onChangeInput }) {
+  return (
+    <>
+      <Label htmlFor={id}>{name}</Label>
+      <input
+        id={id}
+        name={id}
+        type="text"
+        value={value}
+        onChange={(event) => onChangeInput(event)}
```

Comment:
this should work just as good:

```suggestion
        onChange={onChangeInput}
```